src/Disp.js

```diff
@@ -2498,6 +2500,23 @@ CM.Disp.AddMissingUpgrades = function() {
 	}
 }
 
+/**
+ * This function handles custom drawing for the Game.Draw() function.
+ * It is hooked on 'draw' by CM.RegisterHooks()
+ */
+CM.Disp.Draw = function () {
```

Comment:
In the latest revision of CM.Disp I have added a section for `Functions related to display of the full page and initialization of the page`. Perhaps we should rename this to be something like `General functions` and add this function there? I can imagine us adding other things to `CM.Disp.Draw` and then it shouldn't be in the stats page section.

Reply:
I've changed the section header to "General functions related to display, drawing and initialization of the page", I thought "General functions" was really to generic (eh) and that adding "drawing" was enough to include the new `CM.Disp.Draw` method. (Feel free to tell me if you prefere another wording)

I've also moved the aforementioned method into the section

Reply:
Section title is perfect!
Only thing I noticed is that a section called "bottom bar" or something has been removed, so our general section now includes both general as well as bottom bar functions. Probably something I messed up myself while moving the functions around.

To make the document 100% we should move `CM.Disp.Draw` above the bottom bar functions and re-instate a bottom bar section. If you're willing to do that, please go ahead! Otherwise I will do this once I get back to my computer. 
_(I approved this merge since it does fix the issue and I felt that this little reorganization deserves its own pull request)_

Reply:
You're right, I didn't notice that!
This is fixed by #420 (both moving the method and restoring the bottom bar section comment)